### PR TITLE
 Remove CHPL_NIGHTLY_CRON_RECIPIENT from the test scripts

### PR DESCRIPTION
**Pull request by bhavanijayakumaran**
Address https://github.com/Cray/chapel-private/issues/5606
Remove nightly email env from the test script.

### CODE REVIEW

**Review by tzinsky — APPROVED** [review]

File: doc/rst/developer/bestPractices/NightlyTesting.rst
Comment (riftEmber) [inline doc/rst/developer/bestPractices/NightlyTesting.rst:1]: There is another reference to `CHPL_NIGHTLY_CRON_RECIPIENT` on line 19

**Review by riftEmber — CHANGES_REQUESTED** [review]
`util/buildRelease/testRelease` also contains use of `CHPL_NIGHTLY_CRON_RECIPIENT` which should be removed

File: doc/rst/developer/bestPractices/NightlyTesting.rst
Comment (bhavanijayakumaran) [reply doc/rst/developer/bestPractices/NightlyTesting.rst:1]: Removed

**Comment by bhavanijayakumaran** [comment]
> There is still the uses of the variable in `util/buildRelease/testRelease` that should be removed -- could use grep to avoid missing any

Removed all the references in commit #4

**Pull request merged**